### ISSUE TITLE
[Supporting the Platform- Bot Framework Web Chat - Animationcard]: Play/Pause functionality is not provided for animated content present under 'Animation card'.

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5373]
### Is it an issue related to Adaptive Cards?

No.

### User Experience: 
Moving content that starts playing automatically when scrolling the page or any other auto-updating content should be provided a method for low vision, limited mobility also for keyboard-dependent users to pause, stop, hide to control the moving, changing, or frequency of the update in the content.
### Note: 
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Suggested action, Others or unrelated

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type animationcard word in type your message text box.
4. Navigate to the search results.
5. Verify the issue.

### What do you expect?

Play/Pause functionality should be provided for animated content present under 'Animation card'.

### What actually happened?

Play/Pause functionality is not provided for animated content present under 'Animation card'.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/932b576a-2d7e-4a1f-97f8-7f6afd71fdfe)
https://github.com/user-attachments/assets/7b6afeee-78b8-4b3a-a96b-66ed04f195a9

### MAS reference

MAS 2.2.2 – Pause, Stop, Hide

### WCAG reference

 2.2.2 – Pause, Stop, Hide

**Comment by manojs5** [comment]
#A11yMAS;#A11ySev3;#Accessibility;#AILimited;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#MAS2.2.2;#Win11-Edge(Chromium);

**Comment by OEvgeny** [comment]
Animations on the web do not have such functionality.
